src/main/java/reposense/util/FileUtil.java

```diff
@@ -175,6 +177,13 @@ public static void copyTemplate(InputStream is, String outputPath) throws IOExce
         FileUtil.unzip(is, Paths.get(outputPath));
     }
 
+    /**
+     * Copies assets from {@code sourcePath} to the {@code outputPath}.
+     * @throws IOException if {@code is} refers to an invalid path.
+     */
+    public static void copyDirectoryContents(String sourcePath, String outputPath) throws IOException {
+        FileUtils.copyDirectory(new File(sourcePath), new File(outputPath));
```

Comment:
It may be better to write our own `copyDirectoryContents` since the method itself is not so complicated.